src/components/MetaData/index.test.tsx

```diff
@@ -18,1214 +18,1230 @@ jest.mock('react-dom/server', () => ({
 //   dangerouslyInjectJS: [],
 // }));
 
+interface MetaValuesReturnInterface {
+  (prop: string): string | null;
+}
+
+interface GlobalContentBag {
+  [x: string]: object;
+}
+
 const websiteName = 'The Sun';
+const websiteDomain = 'http://example.com';
 const twitterUsername = 'the-sun';
 const resizerURL = 'https://fake.cdn.com/resizer';
+const arcSite = 'the-sun';
+const facebookAdmins = '1111111111';
+const globalContentComplete = {
+  description: {
+    basic: 'this is a description',
+  },
+  headlines: {
+    basic: 'this is a headline',
+  },
+  taxonomy: {
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    seo_keywords: [
+      'keyword1',
+      'keyword2',
+    ],
+    tags: [
+      { slug: 'tag1' },
+      { slug: 'tag2' },
+    ],
+  },
+  // eslint-disable-next-line @typescript-eslint/camelcase
```

Comment:
Since this is applied in a few places might as well apply it to the whole file

Reply:
yes, but is mostly applied over json definitions. Disabling on the whole file could hide other warnings on code.